component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -3013,4 +3017,19 @@ private boolean isEmailOTPAsFirstFactor(AuthenticationContext context) {
 
         return (context.getCurrentStep() == 1 || isPreviousIdPAuthenticationFlowHandler(context));
     }
+
+    /**
+     * This method is to check whether the user is resolved or not.
+     * @param context
```

Comment:
Improve the comment

Reply:
Resolved from https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/162/commits/92e31022d2446de4306ec4ef0fdaada94f4160c1